dvc/repo/diff.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import unicode_literals
+
+import os
+import humanize
+import inflect
+
+
+from dvc.scm.base import FileNotInCommitError
+
+
+def _extract_output(self, output):
```

Comment:
probably not the best name ... took me a few cycles to understand that this is about directories